### PR TITLE
Allow use of merge-after with personal repos

### PR DESCRIPTION
**Pull request by kesyog**
### Summary of changes

 Tweaked merge-after PR URL regex to also allow pull requests on
 personal repos i.e. in the /users/ space

 ### Testing

 * Tested merge-after with a personal repo using a merge trigger in a
 description
 * Checked that merge-after still works with a project repo using a merge
 trigger in a description

### CODE REVIEW

File: polly-merge.py [hunk 1]
```diff
@@ -208,7 +208,7 @@ def get_all_comments(self, projectkey, repositoryslug, pullrequestid):
     def is_pr_merged(self, pr_url_stem):
         """
         Check if a specific PR is merged
-        Returns True if the specificed PR is merged False otherwise
+        Returns True if the specified PR is merged False otherwise
```
Comment (noahp) [inline polly-merge.py:211]: :bow:

File: polly-merge.py [hunk 2]
```diff
@@ -300,10 +301,10 @@ def merge_after(match):
 
         # dictionary of regex-pattern: command to run on match
         commands = {
-            # command: merge
-            f"^{merge_trigger} merge$": just_merge,
             # command: merge-after <url>
-            f"^{merge_trigger} merge-after (.*)$": merge_after,
+            f"^{merge_trigger} after (.*)$": merge_after,
+            # command: merge
+            f"^{merge_trigger}( merge)?$": just_merge,
```
Comment (noahp) [inline polly-merge.py]: Why this change?

Comment (kesyog) [reply polly-merge.py]: :killme: I need to stop being lazy and actually use branches.

This was a local fix that accidentally got added. Y'all broke my merge trigger with your fancy commands. brb

**Comment by noahp** [comment]
Sweet, looks good! thanks man
